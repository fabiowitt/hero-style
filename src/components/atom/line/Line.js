import React from 'react';

import css from './Line.module.scss';

const Line = () => <div className={css['c-line']} />;

Line.propTypes = {};
Line.defaultProps = {};
Line.displayName = 'Line';

export default Line;
