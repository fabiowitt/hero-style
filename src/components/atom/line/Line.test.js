import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Line from './Line';

describe('Line', () => {
  it('should match with the snapshot', () => {
    const wrapper = shallow(<Line />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
