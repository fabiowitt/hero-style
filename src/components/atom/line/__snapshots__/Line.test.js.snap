// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line should match with the snapshot 1`] = `
<div
  className="c-line"
/>
`;
